extensions/amp-bind/0.1/test/test-bind-integration.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import '../../../amp-carousel/0.1/amp-carousel';
+import {installBindForTesting} from '../bind-impl';
+import {toggleExperiment} from '../../../../src/experiments';
+import {chunkInstanceForTesting} from '../../../../src/chunk';
+import {createFixtureIframe} from '../../../../testing/iframe';
+import {bindForDoc} from '../../../../src/bind';
+import {installDocService} from '../../../../src/service/ampdoc-impl';
+
+describe.configure().retryOnSaucelabs().run('integration amp-bind', () => {
+  let iframe;
+  let bind;
+  const fixture = 'test/fixtures/amp-bind-integrations.html';
+
+  beforeEach(() => {
+    return createFixtureIframe(fixture).then(i => {
+      iframe = i;
+      const ampdocService = installDocService(iframe.win, true);
+      const ampdoc = ampdocService.getAmpDoc(iframe.doc);
+      iframe.ampdoc = ampdoc;
+      chunkInstanceForTesting(iframe.ampdoc);
+      toggleExperiment(iframe.win, 'amp-bind', true, true);
+      bind = installBindForTesting(iframe.ampdoc);
+      return iframe.ampdoc.whenReady();
```

Comment:
There might be some quirk to using it. Ideally we shouldn't need to manually set up AMP services in the fixture.

I'll patch and give this a try tomorrow.